doc/manuals/user/filtering.md

```diff
@@ -10,26 +10,25 @@ As a general rule, filters used in standard operation use a scope
-      <queryContextRequest>                                                                                                         "entities": [
-        <entityIdList>                                                                                                              {
-          <entityId type="myEntType" isPattern="true">                                                                                "type": "myEntityType",
-            <id>.*</id>                                                                                                               "isPattern": "true",
-          </entityId>                                                                                                                 "id": ".*"
-        </entityIdList>                                                                                                             }
-        <attributeList>                                                                                                             ],
-        </attributeList>                                                                                                            "restriction": {
-        <restriction>                                                                                                                 "scopes": [
-          <scope>                                                                                                                       {
-            <operationScope>                                                                                                              "type" : "FIWARE::Filter::foobar",
-              <scopeType>FIWARE::Filter::foobar</scopeType>                                                                               "value" : ...
-              <scopeValue>...</scopeValue>                                                                                              }
-            </operationScope>                                                                                                         ]
-          </scope>                                                                                                                  }
-        </restriction>                                                                                                            }
-      </queryContextRequest>                                                                                                      EOF
-      EOF                                                                                                                     
+      (curl localhost:1026/v1/queryContext -s -S --header 'Content-Type: application/json' --header 'Accept: application/json' -d @- | python -mjson.tool) <<EOF
+      {
+	"entities": [
+        {
+	    type": "myEntityType",
+            "isPattern": "true",
+	    "id": ".*"
+        }
```

Comment:
I think this is because the structure of the document this request is implemented with ... (at the beginning) .It is written as is in the wiki ;
http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_User_and_Programmers_Guide#Filters
But has been corrected  all the requested of this section, due to errors on the JSON format